### PR TITLE
add powerbi example

### PR DESCRIPTION
**Pull request by mirnawong1**
adding power bi examples to explorer data health tile doc

### CODE REVIEW

File: website/docs/docs/collaborate/data-tile.md [hunk 1]
```diff
@@ -72,11 +72,49 @@ Follow these steps to set up your data health tile:
...
+<TabItem value="powerbi" label="PowerBI example">
+
+You can embed the data health tile iFrame in PowerBI using PowerBI Pro Online, Fabric PowerBI, or PowerBI Desktop. 
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/power-bi.png" width="80%" title="Embed data health tile iFrame in PowerBI"/>
+
+Follow these steps to embed the data health tile in PowerBI:
+
+1. Create a dashboard in PowerBI and connect to your database to pull in the data.
+2. Create a new measurement with the iFrame.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/data-tile.md]: what does this mean specifically? is this something done in powerbit or explorer?

File: website/docs/docs/collaborate/data-tile.md [hunk 2]
```diff
@@ -72,11 +72,49 @@ Follow these steps to set up your data health tile:
...
+        "<iframe src='https://1234.metadata.us1.dbt.com/exposure-tile?uniqueId=exposure.jaffle_shop.OrderQualityDashboard&environmentType=staging&environmentId=123456789&token=YOUR_METADATA_TOKEN' title='Exposure status tile' height='400'></iframe>"
+    ```
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/power-bi-measure-tools.png" width="100%" title="In the 'Measure tools' tab, replace your values with the iFrame code."/>
+
+6. PowerBI desktop doesn't have HTML rendering by default, so you need to install an HTML visual from the PowerBI Visuals Store.
+7. To do this, go to **Build visuals** and then **Get more visuals**.
+8. Login with your PowerBI account.
+9. Search for an HTML visual and install it. For example, [HTML content](https://appsource.microsoft.com/en-us/product/power-bi-visuals/WA200001930?tab=Overview).
+10. Drag the metric with the iFrame code into the HTML content widget in PowerBI. This should now display your data health tile.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/data-tile.md:110]: is the metric with the iframe code visualized with that calculator-looking icon?

File: website/docs/docs/collaborate/data-tile.md [hunk 3]
```diff
@@ -72,11 +72,49 @@ Follow these steps to set up your data health tile:
...
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/power-bi.png" width="80%" title="Embed data health tile iFrame in PowerBI"/>
+
+Follow these steps to embed the data health tile in PowerBI:
+
+1. Create a dashboard in PowerBI and connect to your database to pull in the data.
+2. Create a new measurement with the iFrame.
+3. In dbt Explorer, navigate to the exposure and expand the [**Embed data health into your dashboard**](/docs/collaborate/data-tile#embed-in-your-dashboard) toggle. 
+4. Go to the **iFrame** tab and copy the iFrame code. Make sure the Metadata Only token is already set up.
+5. In PowerBI, under the **Measure tools** tab, replace your values with the iFrame code in the calculation window.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/data-tile.md]: is it clear where the user should paste the iFrame code?

File: website/docs/docs/collaborate/data-tile.md [hunk 4]
```diff
@@ -72,11 +72,49 @@ Follow these steps to set up your data health tile:
...
+
+    ```html
+        Website =
+        "<iframe src='https://1234.metadata.us1.dbt.com/exposure-tile?uniqueId=exposure.jaffle_shop.OrderQualityDashboard&environmentType=staging&environmentId=123456789&token=YOUR_METADATA_TOKEN' title='Exposure status tile' height='400'></iframe>"
+    ```
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/power-bi-measure-tools.png" width="100%" title="In the 'Measure tools' tab, replace your values with the iFrame code."/>
+
+6. PowerBI desktop doesn't have HTML rendering by default, so you need to install an HTML visual from the PowerBI Visuals Store.
```
Comment (ernestoongaro) [inline website/docs/docs/collaborate/data-tile.md]:
```suggestion
6. PowerBI desktop doesn't support HTML rendering by default, so you need to install an HTML component from the PowerBI Visuals Store.
```

File: website/docs/docs/collaborate/data-tile.md [hunk 5]
```diff
@@ -72,11 +72,49 @@ Follow these steps to set up your data health tile:
...
+7. To do this, go to **Build visuals** and then **Get more visuals**.
+8. Login with your PowerBI account.
+9. Search for an HTML visual and install it. For example, [HTML content](https://appsource.microsoft.com/en-us/product/power-bi-visuals/WA200001930?tab=Overview).
```
Comment (ernestoongaro) [inline website/docs/docs/collaborate/data-tile.md]:
```suggestion
9. There are several third-party HTML visuals. The one tested for this guide is [HTML content](https://appsource.microsoft.com/en-us/product/power-bi-visuals/WA200001930?tab=Overview). Install it, but please keep in mind it's a third-party plugin not created or supported by dbt Labs.
```

Comment (mirnawong1) [reply website/docs/docs/collaborate/data-tile.md]: great shout!

File: website/docs/docs/collaborate/data-tile.md [hunk 6]
```diff
@@ -72,11 +72,55 @@ Follow these steps to set up your data health tile:
...
+
+6. PowerBI desktop doesn't support HTML rendering by default, so you need to install an HTML component from the PowerBI Visuals Store.
+7. To do this, go to **Build visuals** and then **Get more visuals**.
+8. Login with your PowerBI account.
+9. There are several third-party HTML visuals. The one tested for this guide is [HTML content](https://appsource.microsoft.com/en-us/product/power-bi-visuals/WA200001930?tab=Overview). Install it, but please keep in mind it's a third-party plugin not created or supported by dbt Labs.
+10. Drag the metric with the iFrame code into the HTML content widget in PowerBI. This should now display your data health tile.
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/power-bi-final.png" width="80%" title="Drag the metric with the iFrame code into the HTML content widget in PowerBI. This should now display your data health tile."/>
+
+*Refer to [this tutorial](https://www.youtube.com/watch?v=SUm9Hnq8Th8) for additional info on embedding a website into your Power BI report.*
```
Comment (nataliefiann) [inline website/docs/docs/collaborate/data-tile.md]:
```suggestion
*Refer to [this tutorial](https://www.youtube.com/watch?v=SUm9Hnq8Th8) for additional information on embedding a website into your Power BI report.*
```

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

This doc is amazing! Thanks for creating this PR. I made one small change (info to information) but other than that, I have no suggestions or recommendations. I've approved this for you.

Kind Regards
Natalie

**Pull request merged**